frameworks/elastic/docs/tls.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Configuring TLS
+menuWeight: 25
+
+---
+
+# Encrypting Network Communication
+
+Elasticsearch requires
+[some work](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html)
+to get TLS working. The Elastic service takes care of setting everything up when transport encryption is enabled. It will generate and deploy certificates and enable SSL both on the transport and HTTP layers by default.
+
+The following steps describe getting an Elastic service with TLS from scratch. You might already have key pairs, service
+accounts and secrets that you could already use. In that case only run the steps you need.
+
+## Install the Enterprise DC/OS CLI
+
+```
+dcos package install --yes dcos-enterprise-cli
+```
+
+## Create key pair
+
+This generates `elastic.private.pem` and `elastic.public.pem`.
+
+```
+dcos security org service-accounts keypair elastic.private.pem elastic.public.pem
+```
+
+## Create service account
+
+The service account name that is configured for your service (`service.service_account` in the service JSON
+configuration or the `service account` text field in the "Edit Configuration" UI) must match the actual service account
+name, which we're calling `elastic` in this case.
+
+```
+dcos security org service-accounts create -p elastic.public.pem -d "Elastic service account" elastic
+```
+
+## Create service account secret
+
+This stores the private key as a [DC/OS secret](https://docs.mesosphere.com/1.10/security/ent/secrets/) named `elastic-secret`.
```

Comment:
We should use a non-versioned URI for the link. Try replacing `1.10` with `latest` or `current`.